### PR TITLE
Create category heading component

### PR DESCRIPTION
**Pull request by myang1010**
What (if any) features are you implementing?
- Created `CategoryHeading` component with two props, `headingType` and `amount`.

I'm assuming the category heading should go into Programs. For 'Transportation', currently using HealthCare icon.

Screenshots,
- desktop
>desktop

- mobile
>Screenshot 2024-01-18 at 2 29 57 PM

### CODE REVIEW

File: src/Components/Results/Programs/Programs.tsx
Comment (CalebPena) [inline src/Components/Results/Programs/Programs.tsx:27]: Amount is hard coded. @Mdemenko1 created a function to get the value by category in her pr.

File: src/Components/Results/CategoryHeading/CategoryHeading.js
Comment (CalebPena) [inline src/Components/Results/CategoryHeading/CategoryHeading.js:1]: Can we make this file TypeScript.

File: src/Components/Results/CategoryHeading/CategoryHeading.js
Comment (CalebPena) [inline src/Components/Results/CategoryHeading/CategoryHeading.js:11]: @sydney-devine do we have the icon for transportation.

File: src/Components/Results/Programs/Programs.tsx
Comment (CalebPena) [inline src/Components/Results/Programs/Programs.tsx:27]:
```suggestion
            <CategoryHeading headingType={programs[0].category} amount="200" />
```

File: src/Components/Results/CategoryHeading/CategoryHeading.css
Comment (CalebPena) [inline src/Components/Results/CategoryHeading/CategoryHeading.css:1]: We want to put all of the CSS in the `Results.css` file.

**Review by CalebPena — CHANGES_REQUESTED** [review]

File: src/Components/Results/CategoryHeading/CategoryHeading.tsx
Comment (CalebPena) [inline src/Components/Results/CategoryHeading/CategoryHeading.tsx:12]:
```suggestion
const headingOptionsMappings: { [key: string]: React.ComponentType } = {
  'Housing and Utilities': Housing,
  'Food and Nutrition': Food,
  'Health Care': HealthCare,
  Transportation: HealthCare, // placeholder icon
};
```

File: src/Components/Results/CategoryHeading/CategoryHeading.tsx
Comment (CalebPena) [inline src/Components/Results/CategoryHeading/CategoryHeading.tsx:20]:
```suggestion
  const IconComponent = headingOptionsMappings[headingType.default_message];
```

**Review by CalebPena — APPROVED** [review]
Looks good. Will merge when we get the placeholder icon.

File: src/Assets/CategoryHeadingIcons/food.svg
Comment (ivonne-hernandez) [inline src/Assets/CategoryHeadingIcons/food.svg:1]: What do you think about importing these icons from the `AcuteConditions` folder?

File: src/Assets/CategoryHeadingIcons/healthcare.svg
Comment (ivonne-hernandez) [inline src/Assets/CategoryHeadingIcons/healthcare.svg:1]: What do you think about importing these icons from the `AcuteConditions` folder?

File: src/Components/Results/CategoryHeading/CategoryHeading.tsx
Comment (ivonne-hernandez) [inline src/Components/Results/CategoryHeading/CategoryHeading.tsx:11]: Do you still need the actual `Transportation` icon?

**Comment by myang1010** [comment]
Hi @ivonne-hernandez , thanks for the suggestion. I considered reusing icons, but they're scattered. I could only reuse the Food icon from `AcuteConditions`. HealthCare and Housing icons are in different places. I haven't found the Transportation icon in our app. I think it's better to keep all icons together, so I'll hold off the changes for those icons for now.

**Comment by sydney-devine** [comment]
@ivonne-hernandez & @myang1010 See original issue details. I am waiting on our designer for the remainder of the icons that are missing. So transportation, child care, youth, education and (one other one I can't remember off top of head) are coming. I should have them today or early next week so we can swap those icons in where they're missing.

File: src/Components/Results/CategoryHeading/CategoryHeading.tsx
Comment (CalebPena) [inline src/Components/Results/CategoryHeading/CategoryHeading.tsx:15]: We have a `Translation` type for this in `Types/Results.ts`

**Review by CalebPena — APPROVED** [review]

**Pull request merged**
